mchf-eclipse/hardware/mchf_board.c

```diff
@@ -733,68 +733,65 @@ void mchf_board_switch_tx(char mode)
 void mchf_board_power_off(void)
 {
 	ulong i;
-	char	tx[32];
+	char	tx[40];
```

Comment:
Again, one of the strings is longer. I wonder what got corrupted by this call :-)